Gems/ROS2/Code/Source/RobotImporter/RobotImporterWidget.cpp

```diff
@@ -94,13 +94,14 @@ namespace ROS2
                 else
                 {
                     report += "# " + tr("XACRO parsing failed") + "\n";
-                    report += "\n\n## " + tr("Command called") +  "\n\n`" + QString::fromUtf8(outcome.m_called.data()) + "`";
+                    report += "\n\n## " + tr("Command called") + "\n\n`" + QString::fromUtf8(outcome.m_called.data()) + "`";
                     report += "\n\n" + tr("Process failed");
                     report += "\n\n## " + tr("Error output") + "\n\n";
                     report += "```\n";
                     if (outcome.m_logErrorOutput.size())
                     {
-                        report += QString::fromLocal8Bit(outcome.m_logErrorOutput.data(), static_cast<int>(outcome.m_logErrorOutput.size()));
+                        report +=
```

Comment:
It was formatting? If not please remove it.

Reply:
This was caused by my IDEs auto formatting. I'll revert it.